src/main/java/org/mozilla/zest/impl/ZestBasicRunner.java

```diff
@@ -621,6 +621,11 @@ public String getVariable(String name) {
 	}
 
 	@Override
+	public List<String[]> getVariables() {
```

Comment:
From a quick review of the usage of this method in the other pull request I think we should change the `ZestBasicRunner` to allow to set the last request/response instead of exposing the variables. Will add a definitive comment once the review is done.

Reply:
But there is a getter for the variable value. Why not for the keys. :)